conftest.py

```diff
@@ -108,3 +108,23 @@ def _bias_params(evoked, noise_cov, fwd):
     if not mne.forward.is_fixed_orient(fwd):
         want //= 3
     return evoked, fwd, noise_cov, data_cov, want
+
+
+@pytest.fixture(scope="module",
+                params=["mayavi", "vispy"])
+def backend_name(request):
+    yield request.param
+
+
+@pytest.yield_fixture
+def backends_3d(backend_name):
+    from mne.viz import use_test_3d_backend
+    from mne.viz.backends.tests._utils import has_not_vispy
```

Comment:
might be easier to read if the name was `has_vispy` (I don't think we use negation in functions checking for module presence in mne): `if not has_vispy():` instead of `if has_not_vispy():`

Reply:
I agree, I'll modify it